EpsilonLights/Tests/tests/LightsTest/LightsTest.c

```diff
@@ -0,0 +1,373 @@
+    TEST_ASSERT_EQUAL_MESSAGE(1, prevSigState, "prevSigState is supposed to be 1");
+}
+
+/*test blinkSignalLights. left and right signals are both on. 
+prevSigState is 1.*/
+void test_blinkSignalLights_prevSigState1SigLightsLeftONSigLightsRightON()
+{
+    uint32_t prevWakeTime = 0;
+    uint8_t prevSigState = 1;
+    sigLightsHandle.left = 2;
+    sigLightsHandle.right = 2;
+
+    keepBlinkersOFF_resetBlinkerTimerToZero(prevWakeTime, prevSigState);
+    turnBlinkersON_updateTimer(prevWakeTime, prevSigState);
+    keepBlinkersOFF_updateTimer(prevWakeTime, prevSigState);
+}
+
+void keepBlinkersOFF_resetBlinkerTimerToZero(uint32_t prevWakeTime, uint8_t prevSigState)
+{
+    uint32_t blinkerTimer = 771;
+
+    osDelayUntil_ExpectAndReturn(&prevWakeTime, LIGHTS_UPDATE_FREQ, 0);
+    HAL_GPIO_WritePin_Expect(RSIGNAL_GPIO_Port, RSIGNAL_Pin, LIGHT_OFF);
+    HAL_GPIO_WritePin_Expect(LSIGNAL_GPIO_Port, LSIGNAL_Pin, LIGHT_OFF);
+    blinkSignalLights(&prevWakeTime, &blinkerTimer, &prevSigState);
+
+    TEST_ASSERT_EQUAL_MESSAGE(0, blinkerTimer, "blinkerTimer is supposed to be 0");
+}
+
+void turnBlinkersON_updateTimer(uint32_t prevWakeTime, uint8_t prevSigState)
+{
+    uint32_t blinkerTimer = 300;
+    uint32_t expectedBlinkerTimer = blinkerTimer + 10;
+
+    osDelayUntil_ExpectAndReturn(&prevWakeTime, LIGHTS_UPDATE_FREQ, 0);
+    HAL_GPIO_WritePin_Expect(RSIGNAL_GPIO_Port, RSIGNAL_Pin, sigLightsHandle.right);
+    HAL_GPIO_WritePin_Expect(LSIGNAL_GPIO_Port, LSIGNAL_Pin, sigLightsHandle.left);
+    blinkSignalLights(&prevWakeTime, &blinkerTimer, &prevSigState);
+
+    TEST_ASSERT_EQUAL_MESSAGE(expectedBlinkerTimer, blinkerTimer, "blinkerTimer is supposed to be expectedBlinkerTimer");
+}
+
+void keepBlinkersOFF_updateTimer(uint32_t prevWakeTime, uint8_t prevSigState)
+{
+    uint32_t blinkerTimer = 400;
+    uint32_t expectedBlinkerTimer = blinkerTimer + 10;
+
+    osDelayUntil_ExpectAndReturn(&prevWakeTime, LIGHTS_UPDATE_FREQ, 0);
+    HAL_GPIO_WritePin_Expect(RSIGNAL_GPIO_Port, RSIGNAL_Pin, LIGHT_OFF);
+    HAL_GPIO_WritePin_Expect(LSIGNAL_GPIO_Port, LSIGNAL_Pin, LIGHT_OFF);
+    blinkSignalLights(&prevWakeTime, &blinkerTimer, &prevSigState);
+
+    TEST_ASSERT_EQUAL_MESSAGE(expectedBlinkerTimer, blinkerTimer, "blinkerTimer is supposed to be expectedBlinkerTimer");
+}
+
+// TESTS for updateStrobeLight1
+void test_updateStrobeLights1_ESTROBEONUpdateBlinkerTimer()
+{
+    uint32_t prevWakeTime = 0;
+    uint32_t blinkerTimer = 300;
+    char strobeLight;
+    uint32_t expectedBlinkerTimer = blinkerTimer + 10;
+
+    auxBmsInputs[1] = 0b00000001;
+    osDelayUntil_ExpectAndReturn(&prevWakeTime, LIGHTS_UPDATE_FREQ, 0);
+    HAL_GPIO_WritePin_Expect(ESTROBE_GPIO_Port, ESTROBE_Pin, LIGHT_ON);
+    updateStrobeLight(&prevWakeTime, &blinkerTimer, &strobeLight);
+
+    TEST_ASSERT_EQUAL_MESSAGE(expectedBlinkerTimer, blinkerTimer, "blinkerTimer is supposed to be expectedBlinkerTimer");
```

Comment:
Please write exactly what `expectedBlinkerTimer` is suppose to be. A message is meant to provide information to let people know what went wrong and what value it is suppose to be.